packages/backend/src/graphql/resolvers/incoming_payment.ts

```diff
@@ -11,19 +11,20 @@ import {
   errorToCode,
   errorToMessage
 } from '../../open_payments/payment/incoming/errors'
-import { ApolloContext } from '../../app'
+import { TenantedApolloContext } from '../../app'
 import { getPageInfo } from '../../shared/pagination'
 import { Pagination, SortOrder } from '../../shared/baseModel'
 import { GraphQLError } from 'graphql'
 import { GraphQLErrorCode } from '../errors'
 
-export const getIncomingPayment: QueryResolvers<ApolloContext>['incomingPayment'] =
+export const getIncomingPayment: QueryResolvers<TenantedApolloContext>['incomingPayment'] =
   async (parent, args, ctx): Promise<ResolversTypes['IncomingPayment']> => {
     const incomingPaymentService = await ctx.container.use(
       'incomingPaymentService'
     )
     const payment = await incomingPaymentService.get({
-      id: args.id
+      id: args.id,
+      tenantId: ctx.tenant.id
```

Comment:
```suggestion
      tenantId: ctx.isOperator ? undefined : ctx.tenant.id
```
since an operator can request incoming payments for other tenants